config/config-feature-flags.yaml

```diff
@@ -76,10 +76,11 @@ data:
   # Setting this flag to "true" enables CloudEvents for CustomRuns and Runs, as long as a
   # CloudEvents sink is configured in the config-defaults config map
   send-cloudevents-for-runs: "false"
-  # Setting this flag to "enforce" will enforce verification of tasks/pipeline. Failing to verify
-  # will fail the taskrun/pipelinerun. "warn" will only log the err message and "skip"
-  # will skip the whole verification
-  resource-verification-mode: "skip"
+  # Setting this flag to "fail" will enforce verification of tasks/pipeline. Failing to verify
```

Comment:
I think the phrase "enforce verification of tasks/pipeline" is a bit confusing. My understanding is that creating VerificationPolicies is what enforces verification, no? This flag configures behavior only for instances with no matching verificationpolicies. Saying that setting the flag to "ignore" "skips the whole verification" is also confusing, because if you set the flag to "ignore" but there are matching verificationpolicies, the taskrun/pipelinerun could still fail verification right?

Maybe you can rephrase:
```
This flag affects the behavior of taskruns and pipelineruns in cases where no VerificationPolicies match them.
If it is set to "fail", TaskRuns and PipelineRuns will fail verification if no matching policies are found.
If it is set to "warn", TaskRuns and PipelineRuns will run to completion if no matching policies are found, and an error will be logged.
If it is set to "ignore", TaskRuns and PipelineRuns will run to completion if no matching policies are found, and no error will be logged.
```